extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -64,7 +64,8 @@ require([
 		playerInstance.setup({
 			advertising: {
 				autoplayadsmuted: willAutoplay,
-				client: 'googima'
+				client: 'googima',
+				vpaidcontrols: true
```

Comment:
should that be in jwplayer-fandom also?

Reply:
Thanks for notice - https://github.com/Wikia/jwplayer-fandom/pull/4/files